src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task02.java

```diff
@@ -37,6 +37,18 @@ public static void main(String[] args) {
     static String getGreetingByHour(int hour) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String welcome;
+        if (hour >= 0 && hour < 6) {
+            welcome = "Good night";
+        } else if (hour >= 6 && hour < 12) {
+            welcome = "Good morning";
+        } else if (hour >= 12 && hour < 18) {
+            welcome = "Good day";
+        } else if (hour >= 18&& hour < 24) {
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"